### ISSUE TITLE
Note on latitude/longitude numbers

### ISSUE THREAD

**Issue by jofwu** [Palanaeum/roshar-map#69]
Somewhere need to add a note that Rosharans use 200 degrees in a circle. Matters quite a bit for anyone interpreting the numbrs on the latitutde/longitude lines. :-)

Could be written directly on the map?

Could be a text box like the State of the World legend?

Could be a mouseover thing next to the layer option?

**Comment by BlakeGilstrap** [comment]
> Could be a mouseover thing next to the layer option?

Yeah a tooltip on a `fa-info-circle` or similar next to the option would make most sense to me.
